blockly-core/core/ui/block_space/pan_drag_handler.js

```diff
@@ -59,6 +59,19 @@ Blockly.PanDragHandler = function (blockSpace) {
    * @type {BindData}
    * @private
    */
+  this.wheelKey_ = null;
```

Comment:
Would it make sense to name this "wheelKeyEvent_" or "wheelKeyBindData_" to better indicate what sort of thing it is?  wheelKey_ alone sounds like it contains a key code and would require an explanatory comment to explain what it actually is.

Ditto for mousewheelKey_